readthedocs/settings/base.py

```diff
@@ -446,9 +446,6 @@ def USE_PROMOS(self):  # noqa
             'settings': {
                 'number_of_shards': 2,
                 'number_of_replicas': 0,
-                "index": {
-                    "sort.field": ["project", "version"]
```

Comment:
This was giving an error -- https://discuss.elastic.co/t/unable-to-create-index-sorting/166019